package.json

```diff
@@ -49,6 +49,8 @@
     "eslint-plugin-jsx-a11y": "^6.6.1",
     "eslint-plugin-react": "^7.31.11",
     "eslint-plugin-react-hooks": "^4.6.0",
+    "jest": "^27.5.1",
+    "react-test-renderer": "^18.2.0",
```

Comment:
- [ ] One of your tests is failing, kindly update it 
![image](https://user-images.githubusercontent.com/62705947/206228520-5f76536d-315a-4c41-9df5-0cc2dedf5a99.png)

Reply:
Thanks for pointing out the right thing . I found the bugs thanks to you and fix that error . Thanks for helping me improve my code.